examples/render-helm-chart-kustomize-private-git/credentials.env

```diff
@@ -0,0 +1,3 @@
+# This will contain credentials for accessing the private Git repo.
+username=<GH_TOKEN>
```

Comment:
Do we need the placeholders here?

Reply:
Not needed, but thought it might be helpful for us to remember where and how the credentials are stored and generated. Do you prefer we don't include these files?

Reply:
No need to delete it. But I'd suggest adding more comments to make it clear that the values will be populated by the CI and it doesn't work locally.

Reply:
Good idea. Added more details to the comment!